From: Michal Hocko <mhocko@suse.cz>
Subject: kabi: memcg: high limit placeholder
Patch-mainline: never
References: fate#312101

High limit reclaim hasn't been backported as a part of fate#312101
but it might turn out it will be helpful later on so be prepared for
that.

struct mem_cgroup is local to mm/memcontrol.c but people tend to do
crazy things and a single unsigned long as a preparation will cost
us nothing so add a placeholder for the new limit in case we need
it.

The rest of the implementation of the limit can be done in the kabi
safe way.

Signed-off-by: Michal Hocko <mhocko@suse.cz>

---
 mm/memcontrol.c |    1 +
 1 file changed, 1 insertion(+)

--- a/mm/memcontrol.c
+++ b/mm/memcontrol.c
@@ -249,6 +249,7 @@ struct mem_cgroup {
 	struct page_counter kmem;
 
 	unsigned long low_limit;
+	unsigned long high_limit_unused; /* just a placeholder in case we need it later */
 
 	unsigned long soft_limit;
 
